packages/um/package.py

```diff
@@ -150,6 +152,20 @@ class Um(Package):
             "fcm_name": "netcdf",
             "fcm_ld_flags": "-lnetcdff -lnetcdf"}}
 
+    # Source acquisition for AM3
+    resource(
+        name="jules_sources",
+        git="https://github.com/ACCESS-NRI/JULES.git",
+        when="model=\"vn13p1-am\"",
+        destination="."
+    )
+
+    resource(
+        name="um_sources",
+        git="https://github.com/ACCESS-NRI/UM.git",
+        when="model=\"vn13p1-am\"",
+        destination="."
+    )
```

Comment:
How does this define specific versions of the sources?

Reply:
Adding a `tag` argument would help.

Reply:
Resolved in https://github.com/ACCESS-NRI/spack-packages/pull/274/commits/eb9d8d833d5681670bc20dfc703f96d5f88427d9